### DISCUSSION TITLE
Single line output

### DISCUSSION THREAD

**Discussion by sbp-bvanb** [aquasecurity/trivy#5509] [Q&A] [answered]
### Question

For analysis purposes we would like to use a tool that requires single line cdx (json) files. Does Trivy support creation of output files with single line content, e.g.:

{"a":"b"} instead of

{
  "a":"b"
}

Options used: --format cyclonedx in conjunction with --output /some/file

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Version

```bash
0.46.1
```

**Comment by nikpivkin** [comment, marked as answer]
Hi @sbp-bvanb !

You can use `jq` to minify json: `jq -c . < report.json > minified.json`

**Reply by sbp-bvanb** [reply]
Ok so not currently supported in Trivy itself?

**Reply by knqyf263** [reply]
Not supported and planned.
